rllib/agents/trainer_template.py

```diff
@@ -170,9 +132,10 @@ def _init(self, config: TrainerConfigDict,
                 policy_class=self._policy_class,
                 config=config,
                 num_workers=self.config["num_workers"])
-            self.execution_plan = execution_plan
-            self.train_exec_impl = execution_plan(
-                self.workers, config, **self._kwargs_for_execution_plan())
+            if execution_plan is not None:
```

Comment:
`default_execution_plan()` was moved into `Trainer`, so a value of None is ok here.

Reply:
actually can you just add this as a comment:

Override the default_execution_plan set in Trainer.

Reply:
done